src/olympia/lib/crypto/signing.py

```diff
@@ -122,7 +122,7 @@ def call_signing(file_obj):
             # "security.signed_app_signatures.policy"
             # where it's set to COSEAndPKCS7WithSHA1OrSHA256 to match
             # these settings.
-            'pkcs7_digest': 'SHA1',
+            'pkcs7_digest': 'SHA256',
```

Comment:
Need to update the comment above

Reply:
Is there anything relevant about this comment anymore? If we no longer care about versions < 58 and also never use sha1, there is no longer a compatibility issue or a pref to be concerned with, or? Do you have a comment you'd rather have here?

Reply:
Wdyt?

Reply:
Your update from https://github.com/mozilla/addons-server/pull/22408/commits/53a742c2fc8f65233c514f147b013d3223d7cbb4 is fine with me - I just want things to be consistent